content/reference/endpoints/rentals.md

```diff
@@ -152,6 +154,7 @@ initial_price                   | Integer  | Read       | Rental's initial price
 lat                             | Float    | Read/Write | Rental's latitude. (between -90 and 90)
 lng                             | Float    | Read/Write | Rental's longitude. (between -180 and 180)
 max_price                       | [Decimal](/reference/enums#formats)  | Read       | Rental's maximum price.
+mid_term_pricing_active         | Boolean  | Read       | Enable mid term pricing feature for the rental.
```

Comment:
ins't it Write as well?

Reply:
Also not sure how important that is but it may make sense to update response json files